snappi_ixnetwork/snappi_convergence_api.py

```diff
@@ -97,6 +103,10 @@ def set_state(self, payload):
             self._api._connect()
             if payload.choice is None:
                 raise Exception("state [transmit/ link/ route] must configure")
+            if self._remote_link_failure is True:
```

Comment:
Raise exception only when route or link state is set.

Reply:
Add check